common/lib/capa/capa/tests/test_capa_problem.py

```diff
@@ -397,6 +397,26 @@ def test_optionresponse_xml_compatibility(self):
         problem = new_loncapa_problem(xml.format(correctness=False))
         assert problem is not None
 
+    def test_optionresponse_option_with_empty_text(self):
+        """
+        Verify that an optionresponse problem with an option with empty
+        text does fail to instantiate
```

Comment:
`text does not fail to instantiate` ? Thanks for adding this!

Reply:
Fixed.